src/appellate/appellate.js

```diff
@@ -86,7 +98,7 @@ AppellateDelegate.prototype.attachRecapLinksToEligibleDocs = async function () {
 
 AppellateDelegate.prototype.handleDocketDisplayPage = async function () {
   // retrieve pacer_case_id from URL query parameters
```

Comment:
This comment is out of date. I think you can probably just remove it, right? The code is really clear about what it's doing.

Reply:
You're right. Thanks

Changed